numpy/f2py/tests/test_return_integer.py

```diff
@@ -29,8 +29,8 @@ def check_function(self, t, tname):
         pytest.raises(IndexError, t, [])
         pytest.raises(IndexError, t, ())
 
-        pytest.raises(Exception, t, t)
-        pytest.raises(Exception, t, {})
+        pytest.raises(Exception, t, t)   # noqa: B017
+        pytest.raises(Exception, t, {})  # noqa: B017
```

Comment:
These changes seem unrelated? Also in the next f2py test file below. If so, best to drop them from this PR.